### PR TITLE
OSDOCS-4275: Updating future removal section for 4.13 removals

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.12

Issue:
https://issues.redhat.com/browse/OSDOCS-4275

Link to docs preview:
https://51196--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-future-removals

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Comment by bergerhoffer** [comment]
@wangke19 @sanchezl Can you please review? 

Preview: https://51196--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-future-removals

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -349,9 +349,9 @@ For more information, see xref:../cicd/jenkins/important-changes-to-openshift-je
 [id="ocp-4-12-future-removals"]
 === Future Kubernetes API removals
 
-The next minor release of {product-title} is expected to use Kubernetes 1.25. Currently, Kubernetes 1.25 is scheduled to remove several deprecated `v1beta1` and `v2beta1` APIs.
+The next minor release of {product-title} is expected to use Kubernetes 1.26. Currently, Kubernetes 1.26 is scheduled to remove a `v1beta1` and a `v2beta2` API.
```
Comment (sanchezl) [inline release_notes/ocp-4-12-release-notes.adoc:352]:
```suggestion
The next minor release of {product-title} is expected to use Kubernetes 1.26. Currently, there are a number of deprecated APIs scheduled to be removed in Kubernetes 1.26.
```
I still think the text: _"a `v1beta1` and a `v2beta2`"_ is odd. It tries to add some specifics, but they are useless without the full API name. Additionally, it is not consistent from release to release, so something more generic is less work in the long run.

**Review by sanchezl — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (bergerhoffer) [reply release_notes/ocp-4-12-release-notes.adoc:352]: Bah, I made that fix on the other release notes PR I had (#51195), but forgot to do it here. Will remove this here too!

**Review by sanchezl — APPROVED** [review]

**Pull request merged**